### PR TITLE
Update CoreFx to rc3-24125-00

### PR DESCRIPTION
**Pull request by eerhardt**
I also updated update-dependencies to work against Latest_Packages.txt instead of just Latest.txt.

@brthor @livarcocc @piotrpMSFT @pranavkm

### CODE REVIEW

File: TestAssets/TestProjects/TestAppWithResourceDeps/project.json [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-    "NETStandard.Library": "1.5.0-rc2-24027",
-    "Microsoft.CSharp": "4.0.1-rc2-24027",
-    "System.Dynamic.Runtime": "4.0.11-rc2-24027",
-    "System.Reflection.Metadata": "1.3.0-rc2-24027",
-    "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24027",
-    "System.Xml.XmlSerializer": "4.0.11-rc2-24027",
+    "NETStandard.Library": "1.5.0-rc3-24125-00",
+    "Microsoft.CSharp": "4.0.1-rc3-24125-00",
+    "System.Dynamic.Runtime": "4.0.11-rc3-24125-00",
+    "System.Reflection.Metadata": "1.2.1-rc3-24125-00",
```
Comment (pranavkm) [inline TestAssets/TestProjects/TestAppWithResourceDeps/project.json]: :clap:

**Comment by pranavkm** [comment]
:shipit:

File: TestAssets/TestProjects/PortableTests/PortableAppWithIntentionalManagedDowngrade/project.json [hunk 2]
```diff
@@ -10,9 +10,9 @@
           "type": "platform",
           "version": "1.0.0-rc3-*"
         },
-        "System.Linq": "4.0.0"
+        "System.Linq": "4.1.0-rc3-24125-00"
```
Comment (eerhardt) [inline TestAssets/TestProjects/PortableTests/PortableAppWithIntentionalManagedDowngrade/project.json]: @pranavkm - I'm going to have to make a quick change here.  This dependency shouldn't be updated.

Comment (eerhardt) [inline TestAssets/TestProjects/PortableTests/PortableAppWithIntentionalManagedDowngrade/project.json]: Ok - we should be good now.  Let's hope for green.

**Comment by eerhardt** [comment]
Closing as this build is broken by System.Reflection.Metadata.

**Pull request closed without merging**